docs/news/index.qmd

```diff
@@ -2,6 +2,163 @@
+- Improved responsiveness of C / C++ editor intelligence features when switching Git branches (#14320)
+- Show an error dialog if R is not found when starting RStudio Desktop on Linux and macOS (#14343)
+- An empty `rstudio-prefs.json` no longer logs spurious JSON parsing errors (rstudio-pro#4347)
+- Help pane's Next and Previous find-in-topic buttons now have meaningful accessible labels [Accessibility] (#14347)
+- Fixed an issue where the RStudio debugger would re-focus a debugged context over-aggressively (#10664)
+- Fixed handling of duplicated calls in debugger when source references not available (#14276)
+- Fixed issue where stepping through lines of code in tryCatch() would provide incorrect debug highlight (#14306)
+- Fixed an issue where PATH modifications in .Renviron / .Rprofile could be lost on macOS (#9815)
+- Fixed an issue where custom headers were not propagated to `available.packages()` when calling `install.packages()` (#14282)
+- Fixed an issue where code highlight in Sweave \Sexpr{} expressions was incorrectly handled (#14382)
+- Fixed an issue where the Edit button in the Viewer pane could fail for Quarto documents (#14325)
+- Fixed an issue where terminal history would show internal commands (#9833)
+- Fixed an issue where RStudio could fail to launch with Conda builds of R (#13184)
+- Fixed an issue where hovering over S4 class names in the Help pane could produce an R error (#13344)
+- Fixed an issue where defining a custom viewer method produced an R error when View was called (#14439)
+- Fixed an issue where Quarto dashboards could not be previewed if their file path contained a single quote (#13900)
+- Fixed an issue where language objects (calls, formulas) were not described in the Environment pane (#14446)
+- Improved some French translations of the UI
+- Fixed an issue where searches in the Help pane could fail for entries like `[<-` (#10975)
+- Fixed an issue where the "Save workspace on exit" preference was ignored in some cases (#14258)
+- Fixed an issue where the NEWS button in the Update Packages dialog could fail to find the package NEWS file in some cases (#12648)
+- Fixed an issue where Tensorflow object completions were not generated in some contexts (#14524)
+
+#### Posit Workbench
+- Fixed an issue where Professional Driver installation could fail on macOS (rstudio-pro#5168)
+- Fixed Copyright date ranges for Workbench Administrator Guide and Workbench User Guide (rstudio-pro#5865)
+- Fixed low-contrast text on Workbench pages [Accessibility] (rstudio-pro#4464)
+- The terminal pane of JupyterLab sessions now respects the user's default shell (rstudio-pro#5484)
+- The Databricks pane now respects the `databricks-enabled=1` setting in the `rserver.conf` file even when running in legacy RStudio Server Pro mode (rstudio-pro#5695)
+- When launching sessions on Slurm, pending or failed sessions will no longer erroneously appear in the Workbench Jobs pane on the home page (rstudio-pro#4846)
```

Comment:
We use "Fixed" to start each sentence and then sometimes we don't. If we are going to use it for like 97% of the entries, we should consider standardizing